lib/modules.ts

```diff
@@ -0,0 +1,38 @@
+import { Express } from "express";
```

Comment:
nit: modules is a slightly confusing name, given https://hacks.mozilla.org/2018/03/es-modules-a-cartoon-deep-dive/

Reply:
renames to utils